PhotoStream/app/src/main/java/com/keenanzucker/photostream/DbHelper.java

```diff
@@ -0,0 +1,70 @@
+package com.keenanzucker.photostream;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import java.util.ArrayList;
+
+/**
+ * Created by keenan on 10/2/15.
+ * Database helper used to save images, retrieve the saved images, and delete images from the SQL database
+ * Database is simple table that holds saved image URLs.
+ */
+public class DbHelper{
```

Comment:
Coder choice here, but it may make more sense to actually put these methods in your DatabaseService class. That way you have one class that will handle all things database. 
Then again I can also see the merits of doing it this way. @keenanzucker want to tell me why you made this choice?